operator/check_operator.go

```diff
@@ -19,7 +19,7 @@ type Option = func(*operatorCheck)
 const defaultScorecardWaitTime = "240"
 
 // NewCheck is a check runner that executes the Operator Policy.
-func NewCheck(image, kubeconfig, indeximage string, opts ...Option) *operatorCheck {
+func NewCheck(image string, kubeconfig io.Reader, indeximage string, opts ...Option) *operatorCheck {
```

Comment:
Nit: Do we also want to re-order the params to make it more readable? 
```suggestion
func NewCheck(image,  indeximage string, kubeconfig io.Reader, opts ...Option) *operatorCheck {
```

Reply:
Hmm I'm open to this. I figured "order of importance", ranking Kubeconfig > Index Image - but I mean... they're both required so maybe that's silly of me 😆

Reply:
I don't think it matters if they're both required. :) I'd opt for readability.

Reply:
Order changed!